tools/ccache: bump to 4.8 and always disable documentation compilation

misaka36 opened the pull request:
Release Notes:
https://ccache.dev/releasenotes.html#_ccache_4_8

Remove useless include target.mk (https://github.com/openwrt/openwrt/commit/b492e69bd2d09aaa2c3c76b52203cbb3527eb2f7)

misaka36 committed:
tools/ccache: bump to 4.8

Release Notes:
https://ccache.dev/releasenotes.html#_ccache_4_8

Signed-off-by: Linhui Liu <liulinhui36@gmail.com>

mpratt14 commented:
IMO removing target.mk include should be a separate commit, and perhaps that commit could clean up the Makefile in some other ways at the same time

mpratt14 commented:
I think the target.mk include is for `CONFIG_BUILD_DOCUMENTATION`

maybe that should be removed too, setting documentation to no

misaka36 committed:
tools/ccache: always disable documentation compilation

Speed up compilation.

Also remove useless include target.mk
(b492e69bd2d09aaa2c3c76b52203cbb3527eb2f7).

Signed-off-by: Linhui Liu <liulinhui36@gmail.com>
